### PR TITLE
Add UPS test

### PR DESCRIPTION
**Pull request by Stan575**
Added new test for ups.com page as part of Tasks#13 homework. Please review and let me know what you think or find any issues

### CODE REVIEW

File: src/test/java/Group_qui_quaerit_reperit.java [hunk 1]
```diff
@@ -31,4 +33,34 @@ public void oleksandrBurdeinyi() throws InterruptedException {
 
     }
 
+    @Test
+    public void stanAretinskiy() {
+        // Open and maximize browser window
+        WebDriver driver = getDriver();
+        driver.manage().window().maximize();
```
Comment (BELOVEZHIE) [inline src/test/java/Group_qui_quaerit_reperit.java:40]: I don`t have to write this string. On my mind. If you will delete it, be able to see that it works.

**Review by BELOVEZHIE — APPROVED** [review]

**Review by VictoriaRogo — APPROVED** [review]

File: src/test/java/Group_qui_quaerit_reperit.java (hunk 1 above)
Comment (Stan575) [reply src/test/java/Group_qui_quaerit_reperit.java:40]: @BELOVEZHIE thanks for approving. Are you talking about line 40?

Comment (Stan575) [reply src/test/java/Group_qui_quaerit_reperit.java:40]: @BELOVEZHIE I got it, same methods called in BaseTest.java, thanks for bringing it up

Comment (Stan575) [reply src/test/java/Group_qui_quaerit_reperit.java:40]: Removed that line in this branch(SA-Task14), but it's already merged to main, so assuming it's not critical we can leave main as is

**Pull request merged**